src/component-library/NumberInput/NumberInput.tsx

```diff
@@ -1,39 +1,56 @@
+import { useNumberField } from '@react-aria/numberfield';
+import { mergeProps } from '@react-aria/utils';
+import type { NumberFieldStateProps } from '@react-stately/numberfield';
+import { useNumberFieldState } from '@react-stately/numberfield';
 import * as React from 'react';
 
-import { InputProps } from '../Input';
+import { chain } from '@/component-library/utils/chain';
+import { useDOMRef } from '@/component-library/utils/dom';
+
 import { BaseNumberInput } from './NumberInput.style';
 
-// `onWheel` prop can't be used with `preventDefault` because
-// React implements passive event listeners.
-const disableChangeOnWheel = (event: MouseEvent) => {
-  event.preventDefault();
+// Prevents the user from changing the input value using mouse wheel
+const handleWheel = (e: WheelEvent) => e.preventDefault();
```

Comment:
Tiny thing, not a change request - for reasons I can't remember, I've always preferred `event` to `e`. That said, `e` is a more common standard so we should use that consistently. Not a comment for this PR, but as a general rule feel free to refactor `event` to `e` whenever you come across it. Not worth doing as a task on its own though :)

Reply:
@tomjeatt I prefer as well. Done!